packages/block-editor/src/components/rich-text/content.scss

```diff
@@ -40,3 +36,18 @@ figcaption.block-editor-rich-text__editable [data-rich-text-placeholder]::before
 		background: rgb(255, 255, 0);
 	}
 }
+
+[data-rich-text-comment],
+[data-rich-text-format-boundary] {
+	border-radius: 2px;
```

Comment:
Why not $radius-small like before?

Reply:
Hm, I didn't see that, maybe it was changed and missed it in a rebase 🤔